acinclude/squid-util.m4

```diff
@@ -238,7 +238,7 @@ AS_IF([m4_translit([test "x$with_$1" != "xno"], [-+.], [___])],[
     AC_MSG_NOTICE([Library '$1' support: no (auto)])
   ])
 ])
-AM_CONDITIONAL(m4_toupper(m4_translit([ENABLE_LIB$1], [-+.], [___])),[test "x$with_$1" != "xno"])
+AM_CONDITIONAL(m4_toupper(m4_translit([ENABLE_LIB$1],[-+.],[___])),m4_translit([test "x$with_$1" != "xno"],[-+.],[___]))
```

Comment:
This change seems out of scope

Reply:
This is already in master, just bringing the branch up to sync.